### ISSUE TITLE
game bugs 3

### ISSUE THREAD

**Issue by DavidWhaleMEF** [Microsoft/pxt#1923]
microbit.org-ticket: 1268

- once you create a sprite even if you delete it - you cannot draw on the display, write text or manipulate pixels until a game over is issued (or better say - until the micro:bit has been reset) - so all the sprites are removed but still there is a layer overlaying the display

**Comment by DavidWhaleMEF** [comment]
Customer writing us a repro .hex at moment

**Comment by tballmsft** [comment]
@DavidWhaleMEF: Always nice to get JavaScript code pasted in the issue.

**Comment by pelikhan** [comment]
Was due to double registeration of sprites.
```
let item1 = game.createSprite(1, 1)
input.onButtonPressed(Button.A, () => {
    item1.delete();
})
input.onButtonPressed(Button.B, () => {
    let item2 = game.createSprite(2, 2)
})
```

**Comment by pelikhan** [comment]
Fix in 0.11.10.
